### DISCUSSION TITLE
Compile from source with static runtime directory?

### DISCUSSION THREAD

**Discussion by ardrigh** [helix-editor/helix#6580] [General]
Compiling from source creates a symlink from my `helix` git clone directory to `~/.config/helix/runtime` which seems like it might break if Fedora gets an official package to install via dnf.

Is there a way to compile from source and specify the runtime directory, so it's not a symlink?

```shell
> hx --health
Config file: default
Language file: default
Log file: /home/user/.cache/helix/helix.log
Runtime directories: /home/user/.config/helix/runtime;/home/user/.cargo/bin/runtime
Runtime directory /home/user/.config/helix/runtime is symlinked to: /home/user/workspace/helix/runtime
Runtime directory does not exist: /home/user/.cargo/bin/runtime
Clipboard provider: xclip
System clipboard provider: xclip
```

I can look at making my own rpm spec file if there is more build documentation

**Comment by CptPotato** [comment]
You can override the runtime directory using the `HELIX_RUNTIME` environment variable ([see manual](https://docs.helix-editor.com/master/install.html?#configuring-helixs-runtime-files)).
